### PR TITLE
fix: anchor tag inside anchor tag on squad shared post

### PR DESCRIPTION
**Pull request by omBratteng**
### Describe what this PR does
- fix: anchor tag inside anchor tag on squad shared post
  Removes wrapping anchor tag and adds anchor tag for title and image

## Events

N/A

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/post/SharePostContent.tsx [hunk 1]
```diff
@@ -12,6 +12,21 @@ import SettingsContext from '../../contexts/SettingsContext';
...
+interface PostLinkProps {
+  href: string;
+  as?: string;
+  className?: string;
+  children: React.ReactNode;
+}
+
+const PostLink = ({ href, as, className, children }: PostLinkProps) => {
+  return (
+    <Link href={href} as={as}>
```
Comment (rebelchris) [inline packages/shared/src/components/post/SharePostContent.tsx:24]: I think the target is missing from what it used to be.
For external is should be _blank

Comment (rebelchris) [reply packages/shared/src/components/post/SharePostContent.tsx:24]: Same with rel

**Review by rebelchris — CHANGES_REQUESTED** [review]
Small fix needed

**Review by rebelchris — APPROVED** [review]
Good from my side, lets see what @capJavert says

**Review by capJavert — APPROVED** [review]
I think its fine for this case.

**Pull request merged**